web/themes/custom/liibarrio2020/liibarrio2020.libraries.yml

```diff
@@ -8,6 +8,7 @@ global-styling:
   css:
     component:
       css/main.css: {}
+      css/custom.css: {}
```

Comment:
This will cause the user to download both css files. The preferred way of adding CSS is to add it as an additional SCSS file, which is then compiled into main.css. That means (a) the user only has to download one file (b) you get to use SCSS and Bootstrap variables and (c) the SCSS can be grouped based on what it does.

See https://github.com/africanlii/liiweb#changing-css-and-js-in-the-liibarrio2020-theme for quick details on how to do this. Take a look at main.scss and you'll see how it imports from a number of other files. 

In this case, I think these changes could easily go into `components/_view.scss`, which is where other similar changes sit.